app/Services/Media.php

```diff
@@ -229,12 +229,11 @@ public function getInfo(SplFileInfo $file)
             return;
         }
 
-        $track = array_get($info, 'comments.track_number', [0])[0];
+        $track = array_get($info, 'comments.track', [0])[0] ?: array_get($info, 'comments.tracknumber', [0])[0] ?: array_get($info, 'comments.track_number', [0])[0] ?: 0;
```

Comment:
Please make this more readble. Ternary operators here only make things worse IMO.